content/en/docs/tutorials/stateless-application/guestbook-logs-metrics-with-elk.md

```diff
@@ -8,6 +8,8 @@ card:
   name: tutorials
   weight: 31
   title: "Example: Add logging and metrics to the PHP / Redis Guestbook example"
+# setting to draft as this needs to be rewritten/updated for Mongo
+draft: true
```

Comment:
Please do that rewriting.

Reply:
Alternatively: remove it. Git always lets us get it back.

Reply:
removed!